src/waveform/renderers/waveformmark.h

```diff
@@ -21,6 +21,14 @@ class WaveformRenderMark;
 
 class WaveformMark {
   public:
+    class Graphics {
+      public:
+        Graphics()
+                : m_obsolete{false} {
+        }
+        bool m_obsolete;
```

Comment:
I think this is equivalent and simpler, though we might have a style reason not to use this format? 
```suggestion
        bool m_obsolete = false;
```

Reply:
See my comment below on uniform initialization.